api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 246**
+* [`POST /register`](/api/register-queue), [`POST /events`](/api/get-events):
```

Comment:
You're missing a blank line after the feature level. This does not work, and also shows you're not testing your API documentation changes by actually viewing them on localhost:9991/api. Please do that -- you should be manually testing correctness for every line of code you write, especially documentation -- it's really really easy to have documentation that's busted if you don't test, and looks very unprofessional :)

![image](https://github.com/zulip/zulip/assets/2746074/f6f1ad15-03b9-49a7-a922-ea44700bc1a7)

Reply:
sorry! you're right, I never actually tested the /api till now... I'll make sure to do it from now on and update the remaining PRs as well.